package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "scrape2md",
   "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "description": " A package to scrape web content into readable markdown for llms and human readers",
+  "main": "./src/index.ts",
```

Comment:
You need to build this to JS.  I would recommend looking at what Yumei did with the other Typescript2Openai repo.